internal/support/trace_parser.py

```diff
@@ -469,6 +469,22 @@ def ProcessTimelineEvents(self):
                 self.interactive.append([int(math.ceil(self.interactive_start / 1000.0)),
                                          int(math.floor(self.interactive_end / 1000.0))])
 
+            # Extend last interactive window to the end of the run
+            # Assuming that if there are no events past the last interactive window
+            # then the page was interactive for that time
+            if self.cpu['total_usecs'] == self.interactive_end and run_time > 0 and \
+                    len(self.interactive):
+                run_time_ms = int(math.floor(run_time * 1000.0))
+                last_window = self.interactive.pop()
+                if last_window[1] != self.cpu['total_usecs']:
+                    new_window = [int(math.ceil(self.cpu['total_usecs'] / 1000.0)),
+                                  int(math.floor(run_time_ms /1000.0))]
```

Comment:
Is this right?  usecs/1000 should result in milliseconds, not seconds.  I don't think we want to divide the run_time_ms by 1000.